Always use wx-config to get the correct paths.
Hardcoding paths is completely useless.
Index: freefilesync-6.11/Makefile
===================================================================
--- freefilesync-6.11.orig/Makefile
+++ freefilesync-6.11/Makefile
@@ -5,7 +5,7 @@ SHAREDIR    = $(DESTDIR)$(prefix)/share
 APPSHAREDIR = $(SHAREDIR)/$(APPNAME)
 DOCSHAREDIR = $(SHAREDIR)/doc/$(APPNAME)
 
-CXXFLAGS  = -std=c++11 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../.. -I../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" -Wall \
+CXXFLAGS  = -std=c++11 -pipe -DWXINTL_NO_GETTEXT_MACRO -I. -include "zen/i18n.h" -include "zen/warn_static.h" -Wall \
 -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -DZEN_LINUX -pthread
 
 LINKFLAGS = -s `wx-config --libs std, aui --debug=no` -lboost_thread -lboost_system -lz -pthread
@@ -67,53 +67,53 @@ CPP_LIST+=lib/perf_check.cpp
 CPP_LIST+=lib/status_handler.cpp
 CPP_LIST+=lib/versioning.cpp
 CPP_LIST+=lib/ffs_paths.cpp
-CPP_LIST+=../../zen/xml_io.cpp
-CPP_LIST+=../../zen/recycler.cpp
-CPP_LIST+=../../zen/file_access.cpp
-CPP_LIST+=../../zen/file_io.cpp
-CPP_LIST+=../../zen/file_traverser.cpp
-CPP_LIST+=../../zen/zstring.cpp
-CPP_LIST+=../../zen/format_unit.cpp
-CPP_LIST+=../../zen/process_priority.cpp
-CPP_LIST+=../../wx+/grid.cpp
-CPP_LIST+=../../wx+/image_tools.cpp
-CPP_LIST+=../../wx+/graph.cpp
-CPP_LIST+=../../wx+/tooltip.cpp
-CPP_LIST+=../../wx+/image_resources.cpp
-CPP_LIST+=../../wx+/popup_dlg.cpp
-CPP_LIST+=../../wx+/popup_dlg_generated.cpp
-CPP_LIST+=../../wx+/zlib_wrap.cpp
+CPP_LIST+=zen/xml_io.cpp
+CPP_LIST+=zen/recycler.cpp
+CPP_LIST+=zen/file_access.cpp
+CPP_LIST+=zen/file_io.cpp
+CPP_LIST+=zen/file_traverser.cpp
+CPP_LIST+=zen/zstring.cpp
+CPP_LIST+=zen/format_unit.cpp
+CPP_LIST+=zen/process_priority.cpp
+CPP_LIST+=wx+/grid.cpp
+CPP_LIST+=wx+/image_tools.cpp
+CPP_LIST+=wx+/graph.cpp
+CPP_LIST+=wx+/tooltip.cpp
+CPP_LIST+=wx+/image_resources.cpp
+CPP_LIST+=wx+/popup_dlg.cpp
+CPP_LIST+=wx+/popup_dlg_generated.cpp
+CPP_LIST+=wx+/zlib_wrap.cpp
 
-OBJECT_LIST = $(CPP_LIST:%.cpp=../Obj/FFS_GCC_Make_Release/ffs/src/%.o)
+OBJECT_LIST = $(CPP_LIST:%.cpp=Obj/FFS_GCC_Make_Release/ffs/src/%.o)
 
 all: launchpad
 
 launchpad: FreeFileSync
 
-../Obj/FFS_GCC_Make_Release/ffs/src/%.o : %.cpp
+Obj/FFS_GCC_Make_Release/ffs/src/%.o : %.cpp
 	mkdir -p $(dir $@)
 	g++ $(CXXFLAGS) -c $< -o $@
 
 FreeFileSync: $(OBJECT_LIST)
-	g++ -o ../Build/$(APPNAME) $(OBJECT_LIST) $(LINKFLAGS)
+	g++ -o Build/$(APPNAME) $(OBJECT_LIST) $(LINKFLAGS)
 
 clean:
-	rm -rf ../Obj/FFS_GCC_Make_Release
-	rm -f ../Build/$(APPNAME)
-	rm -f ../../wx+/pch.h.gch
+	rm -rf Obj/FFS_GCC_Make_Release
+	rm -f Build/$(APPNAME)
+	rm -f wx+/pch.h.gch
 
 install:
 	mkdir -p $(BINDIR)
-	cp ../Build/$(APPNAME) $(BINDIR)
+	cp Build/$(APPNAME) $(BINDIR)
 
 	mkdir -p $(APPSHAREDIR)
-	cp -R ../Build/Languages/ \
-	../Build/Help/ \
-	../Build/Sync_Complete.wav \
-	../Build/Resources.zip \
-	../Build/styles.gtk_rc \
+	cp -R Build/Languages/ \
+	Build/Help/ \
+	Build/Sync_Complete.wav \
+	Build/Resources.zip \
+	Build/styles.gtk_rc \
 	$(APPSHAREDIR)
 
 	mkdir -p $(DOCSHAREDIR)
-	cp ../Build/Changelog.txt $(DOCSHAREDIR)/changelog
+	cp Build/Changelog.txt $(DOCSHAREDIR)/changelog
 	gzip $(DOCSHAREDIR)/changelog
Index: freefilesync-6.11/RealtimeSync/Makefile
===================================================================
--- freefilesync-6.11.orig/RealtimeSync/Makefile
+++ freefilesync-6.11/RealtimeSync/Makefile
@@ -2,7 +2,7 @@ APPNAME = RealtimeSync
 prefix = /usr
 BINDIR = $(DESTDIR)$(prefix)/bin
 
-CXXFLAGS  = -std=c++11 -pipe -DWXINTL_NO_GETTEXT_MACRO -I../../.. -I../../../zenXml -include "zen/i18n.h" -include "zen/warn_static.h" -Wall \
+CXXFLAGS  = -std=c++11 -pipe -DWXINTL_NO_GETTEXT_MACRO -I. -I.. -include "zen/i18n.h" -include "zen/warn_static.h" -Wall \
 -O3 -DNDEBUG `wx-config --cxxflags --debug=no` -DZEN_LINUX -pthread
 
 LINKFLAGS = -s `wx-config --libs std, aui --debug=no` -lboost_thread -lboost_system -lz -pthread
@@ -25,36 +25,36 @@ CPP_LIST+=../lib/localization.cpp
 CPP_LIST+=../lib/process_xml.cpp
 CPP_LIST+=../lib/resolve_path.cpp
 CPP_LIST+=../lib/ffs_paths.cpp
-CPP_LIST+=../../../zen/xml_io.cpp
-CPP_LIST+=../../../zen/dir_watcher.cpp
-CPP_LIST+=../../../zen/file_access.cpp
-CPP_LIST+=../../../zen/file_io.cpp
-CPP_LIST+=../../../zen/file_traverser.cpp
-CPP_LIST+=../../../zen/zstring.cpp
-CPP_LIST+=../../../zen/format_unit.cpp
-CPP_LIST+=../../../wx+/image_tools.cpp
-CPP_LIST+=../../../wx+/image_resources.cpp
-CPP_LIST+=../../../wx+/popup_dlg.cpp
-CPP_LIST+=../../../wx+/popup_dlg_generated.cpp
+CPP_LIST+=../zen/xml_io.cpp
+CPP_LIST+=../zen/dir_watcher.cpp
+CPP_LIST+=../zen/file_access.cpp
+CPP_LIST+=../zen/file_io.cpp
+CPP_LIST+=../zen/file_traverser.cpp
+CPP_LIST+=../zen/zstring.cpp
+CPP_LIST+=../zen/format_unit.cpp
+CPP_LIST+=../wx+/image_tools.cpp
+CPP_LIST+=../wx+/image_resources.cpp
+CPP_LIST+=../wx+/popup_dlg.cpp
+CPP_LIST+=../wx+/popup_dlg_generated.cpp
 
-OBJECT_LIST=$(CPP_LIST:%.cpp=../../Obj/RTS_GCC_Make_Release/ffs/src/rts/%.o)
+OBJECT_LIST=$(CPP_LIST:%.cpp=../Obj/RTS_GCC_Make_Release/ffs/src/rts/%.o)
 
 all: launchpad
 
 launchpad: RealtimeSync
 
-../../Obj/RTS_GCC_Make_Release/ffs/src/rts/%.o : %.cpp
+../Obj/RTS_GCC_Make_Release/ffs/src/rts/%.o : %.cpp
 	mkdir -p $(dir $@)
 	g++ $(CXXFLAGS) -c $< -o $@
 
 RealtimeSync: $(OBJECT_LIST)
-	g++ -o ../../Build/$(APPNAME) $(OBJECT_LIST) $(LINKFLAGS)
+	g++ -o ../Build/$(APPNAME) $(OBJECT_LIST) $(LINKFLAGS)
 
 clean:
-	rm -rf ../../Obj/RTS_GCC_Make_Release
-	rm -f ../../Build/$(APPNAME)
-	rm -f ../../../wx+/pch.h.gch
+	rm -rf ../Obj/RTS_GCC_Make_Release
+	rm -f ../Build/$(APPNAME)
+	rm -f ../wx+/pch.h.gch
 
 install:
 	mkdir -p $(BINDIR)
-	cp ../../Build/$(APPNAME) $(BINDIR)
+	cp ../Build/$(APPNAME) $(BINDIR)
